src/parser/lib/EventRequestParser.php

```diff
@@ -163,181 +50,12 @@ public static function parseEventRequest(string $body, string $channelSecret, st
             throw new InvalidEventRequestException();
         }
 
+        /* @var Event[] $events */
```

Comment:
(optional) Can you comment above `parseEventRequest` what values are returned when an unsupported webhook event arrives in the SDK? It's already been thoroughly tested, so just documenting it would be fine. There are two types, and neither seems to result in an error! (Nice!)

- When the top-level type is unknown (e.g., `"type":"unknown"`)
- When the top-level type is known, but the type of a nested field is unknown (e.g., `"type":"message", "message.type":"unknown"`)

Reply:
thanks!